source/tyr/readme.md

```diff
@@ -740,52 +740,50 @@ If nothing if configured, default POI are extracted. At this time the list is as
 - uri: "leisure:garden" name: "Jardin"
 - uri: "leisure:park" name: "Zone Parc. Zone verte ouverte, pour déambuler. habituellement municipale"
 
-Let say you want to see all type of poi for the instance fr-bre, you have to call the PoiTypes endpoint for this
+Let say you want to see all poi types for the instance fr-bre, you have to call the PoiTypes endpoint for this
 instance:
 
     curl "http://localhost:5000/v0/instances/fr-bre/poi_types"
 ```json
 {
-    "poi_types": [
-        {
-            "name": "foo",
-            "uri": "foo:ba"
-        },
-        {
-            "name": "foo\u00e9",
-            "uri": "foo:baff"
-        }
-    ]
+
+  "poi_types": [
+    {"id": "pdv", "name": "Point de vente"},
+    {"id": "mairie", "name": "Hôtel de ville"}
```

Comment:
We now ask the conf to have at least 2 poi-types with id `amenity:parking` and `amenity:bicycle_rental`.
Not mandatory to use them in rules, but as their use is hard-coded (for the switch with walking in journeys), it would be better if documented.